frontend/src/components/MakerForm/MakerForm.tsx

```diff
@@ -297,6 +310,7 @@ const MakerForm = ({
     if (maker.advancedOptions) {
       handleClickRelative();
       setMaker({ ...maker, advancedOptions: false });
+      setAmountRangeEnabled(false);
```

Comment:
Given that the ultimate control is `makerHasAmountRange`, and if `amountRangeEnabled` is true by default, this one is not needed.  This way the switch state persists when user enables/disables advanced options.

Reply:
Resolved in the last commit 375eb495016decfc2bffd61172e8ab50fce5dc8a.